src/plugins/administrative_templates/ui/platformmodel.cpp

```diff
@@ -19,22 +19,25 @@
 ***********************************************************************************************************************/
 
 #include "platformmodel.h"
+#include "bundle/policyroles.h"
+#include "qnamespace.h"
 
+#include <iostream>
 #include <memory>
+#include <unordered_map>
 
+#include <QDebug>
```

Comment:
We need to add std::hash for QString for this to compile with gcc version 8.4.1 20200305.

Reply:
But at the same time it's redefinition error on gcc 10.3.1 20210703: already defined in `/usr/include/qt5/QtCore/qhashfunctions.h`